docs/how-to-guides/streams/kafka/connect-to-stream-from-lab.md

```diff
@@ -0,0 +1,89 @@
+---
+id: connect-to-stream-from-lab
+title: How to connect to a data stream from Memgraph Lab
+sidebar_label: Connect to a data stream from Memgraph Lab
+---
+
+If you prefere to use GUI, you can now connect to data streams by using a wizard
+in the Memgraph Lab.
+
+If you need a stream to play around with, we've provided some [Awesome
+Streams](https://awesomedata.stream/)! 
+
+## 1. Add a stream
+
+To add a stream in Memgraph Lab: 
+
+1. Switch to **Streams** and **Add New Stream**.
+2. Choose Kafka stream type, enter a stream name, server address, and topics you want to subscribe to.
+3. Go to the **Next Step**.
+
+If you are trying to connect to MovieLens data stream, the stream configuration
+should looks like this:
+
+
+Once the basic configuration is finished, you need to define a transformation
+module and attach it to the stream. 
+
+
+## 2. Add a transformation module
+
+A transformation module is a set of user-defined transformation procedures
+written in C or Python that act on data received from a streaming engine.
+Transformation procedures instruct Memgraph on how to transform the incoming
+messages to consume them correctly. 
+
+To add the transformation module:
+1. Click on **Add Transformation Module**.
+2. Click on **Choose Transformation Module**.
+3. Select an existing transformation module or **+ Create new transformation**.
+4. Review an existing module or clear the screen and write a new transformation
+   procedure.
+5. Save the transformation module.
+6. Check if the necessary transformation procedure is visible under **Detected
+   transformation functions** on the right.
+7. Select a transformation procedure and **Attach to Stream**.
+
+Check the transformation module for MovieLens on [Awesome Streams](https://awesomedata.stream/#/movielens).
+
+## 3. Set Kafka configuration parameters
+
+If necessary, add the Kafka configuration parameters to customize the stream further:
+
+1. In the Kafka Configuration Parameters **+ Add parameter field**.
+2. Insert the parameter name and value.
+3. To add another parameter, **Add parameter filed**.
+4. Save Configuration once you have set all parameters.
+
+You can also customize the *Batch Interval* and *Batch Size* by editing the basic stream configuration.
+
+To connect to the [Awesome Streams](https://awesomedata.stream/) you need to set
+the following Kafka configuration parameters:
+
+sasl.username:`public`
+sasl.password:`public`
+security.protocol:`SASL_PLAINTEXT`
+sasl.mechanism:`PLAIN`

```

Comment:
The rendering of this part doesn't look good on the front end. Maybe add it into the code block with no copy button?

Reply:
I added <br> Is ti ok now?
 I don't want to put it as code because you enter it in GUI, so it's not really a code